atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -234,11 +238,15 @@ private void runOnceForTable(TableToSweep tableToSweep) {
                     batchConfig,
                     startRow);
             long elapsedMillis = watch.elapsed(TimeUnit.MILLISECONDS);
-            log.debug("Swept {} unique cells from {} starting at {}"
+            log.info("Swept {} unique cells from {} starting at {}"
                             + " and performed {} deletions in {} ms"
                             + " up to timestamp {}.",
-                    results.getCellTsPairsExamined(), tableRef, startRowToHex(startRow),
-                    results.getStaleValuesDeleted(), elapsedMillis, results.getSweptTimestamp());
+                    SafeArg.of("", results.getCellTsPairsExamined()),
+                    UnsafeArg.of("table name", tableRef),
+                    SafeArg.of("row hex", startRowToHex(startRow)),
+                    SafeArg.of("", results.getStaleValuesDeleted()),
+                    SafeArg.of("", elapsedMillis),
+                    SafeArg.of("", results.getSweptTimestamp()));
```

Comment:
per internal discussions, we're leaning towards not doing substitution, so names are critical